main.go

```diff
@@ -61,6 +61,7 @@ var configDefaults = map[string]interface{}{
 	"name":                                 "",
 	"secret":                               "",
 	"token_hmac_secret_key":                "",
+	"jwks_public_url":                      "",
```

Comment:
`token_jwks_public_url` is better to be more consistent with existing option names

Reply:
`token_jwks_public_endpoint` even better to match our proxy options that also have endpoints configurable

Reply:
Good.